pkgs/development/python-modules/zerobin/default.nix

```diff
@@ -1,24 +1,28 @@
 { stdenv
 , buildPythonPackage
 , fetchFromGitHub
-, cherrypy
+, appdirs
+, beaker
+, bleach
 , bottle
-, lockfile
+, cherrypy
 , clize
+, lockfile
+, paste
 }:
```

Comment:
🐛  Running the built executable failed in my environment:

```
/nix/store/dk6rznsq95h6m47nvcrjvxbg74qzj1mm-python3.8-zerobin-1.0.5/bin/zerobin
Traceback (most recent call last):
  File "/nix/store/dk6rznsq95h6m47nvcrjvxbg74qzj1mm-python3.8-zerobin-1.0.5/bin/.zerobin-wrapped", line 6, in <module>
    from zerobin.cli import main
  File "/nix/store/dk6rznsq95h6m47nvcrjvxbg74qzj1mm-python3.8-zerobin-1.0.5/lib/python3.8/site-packages/zerobin/__init__.py", line 3, in <module>
    __version__ = (Path(__file__).parent / "VERSION").read_text().strip()
  File "/nix/store/z65l1jqvxa58zzwwa3bvglb6asj4y8cv-python3-3.8.5/lib/python3.8/pathlib.py", line 1232, in read_text
    with self.open(mode='r', encoding=encoding, errors=errors) as f:
  File "/nix/store/z65l1jqvxa58zzwwa3bvglb6asj4y8cv-python3-3.8.5/lib/python3.8/pathlib.py", line 1218, in open
    return io.open(self, mode, buffering, encoding, errors, newline,
  File "/nix/store/z65l1jqvxa58zzwwa3bvglb6asj4y8cv-python3-3.8.5/lib/python3.8/pathlib.py", line 1074, in _opener 
    return self._accessor.open(self, flags, mode)
FileNotFoundError: [Errno 2] No such file or directory: '/nix/store/dk6rznsq95h6m47nvcrjvxbg74qzj1mm-python3.8-zerobin-1.0.5/lib/python3.8/site-packages/zerobin/VERSION'
```

Running the pydoit build script in the build phase seems to have fixed it:

<details><summary>diff</summary>


```diff
diff --git a/pkgs/development/python-modules/zerobin/default.nix b/pkgs/development/python-modules/zerobin/default.nix
index 6c80226bd68..03a95c75364 100644
--- a/pkgs/development/python-modules/zerobin/default.nix
+++ b/pkgs/development/python-modules/zerobin/default.nix
@@ -7,8 +7,11 @@
 , bottle
 , cherrypy
 , clize
+, doit
 , lockfile
 , paste
+, pyscss
+, nodePackages
 }:
 
 buildPythonPackage {
@@ -22,8 +25,13 @@ buildPythonPackage {
     sha256 = "sha256-JvgPWkycHJApHZJcvf3Oxm0fq625jr0FJ5/6IRAc3rU";
   };
 
+  nativeBuildInputs = [ doit pyscss nodePackages.uglify-js ];
   propagatedBuildInputs = [ appdirs beaker bleach bottle cherrypy clize lockfile paste ];
 
+  buildPhase = ''
+    doit build
+  '';
+
   # zerobin doesn't have any tests, but includes a copy of cherrypy which
   # can wrongly fail the check phase.
   doCheck = false;
```

</details>